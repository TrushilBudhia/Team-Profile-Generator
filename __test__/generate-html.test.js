const generateHtml = require('../lib/generate-html');

describe('generate-html', () => {
    it('No employee cards are displayed when the team contains 0 members', () => {
        expect(generateHtml([])).toMatchSnapshot();
    });
});