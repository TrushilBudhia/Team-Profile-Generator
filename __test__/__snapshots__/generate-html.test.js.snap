// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate-html No employee cards are displayed when the team contains 0 members 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
  <head>
    <meta charset=\\"UTF-8\\" />
    <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\" />
    <meta http-equiv=\\"X-UA-Compatible\\" content=\\"ie=edge\\" />
    <title>Team Profile</title>
    <link
      href=\\"https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css\\"
      rel=\\"stylesheet\\"
    />
    <link rel=\\"stylesheet\\" href=\\"./style.css\\" />
  </head>

  <body>
    <header
      class=\\"
        bg-gradient-to-r
        from-red-500
        to-blue-800
        flex
        justify-center
        max-w-full
        p-7
      \\"
    >
      <section class=\\"team-heading\\">
        <h1 class=\\"font-serif text-6xl text-white\\">â™”My Team</h1>
      </section>
    </header>
    <main class=\\"flex items-center justify-center\\">
      <section
        class=\\"
          team-area
          grid grid-cols-1
          gap-10
          sm:grid-cols-1
          md:grid-cols-2
          lg:grid-cols-3
          xl:grid-cols-3
          my-8
          mx-5
        \\"
      ></section>
    </main>
  </body>
</html>
"
`;
